index.html

```diff
@@ -0,0 +1,233 @@
+          </p>
+        </div>
+        <div class="vedate">
+          <h4 class="date">2023.03.08(WED)~11(SAT)</h4>
+          <p class="venue">
+            @ The Kenyatta International Convention Centre(KICC), Nairobi-Kenya
+          </p>
+        </div>
+      </div>
+    </header>
+
+    <main class="main">
+      <!--Main Program -->
+      <section class="program">
+        <div class="program-info">
+          <div class="pro-heading">
+            <h3 class="pro-text">Main Program</h3>
+            <div class="underline-orange"></div>
+            <hr class="line" />
+          </div>
+          <div class="procard">
+            <article class="program-container">
+              <div class="left-text">
+                <img class="leficon" src="./image/program_icon_01.png" alt="lecture icon" />
+                <p class="leftext">The New Business Model</p>
+              </div>
+              <div class="right-text">
+                <p class="rightext">
+                  It will be characterized by the interdependence, rather
+                  than independence, of its parts, while insuring 
+                  that each part of the firm is a contributor to value.
+                </p>
+              </div>
+            </article>
+            <article class="program-container">
+              <div class="left-text">
+                <img class="leficon" src="./image/program_icon_02.png" alt="program icon" />
+                <p class="leftext">Open Innovation & Next-Gen Partnering</p>
+              </div>
+              <div class="right-text">
+                <p class="rightext">
+                  You will learn more on a practice of businesses that actively
+                  share knowledge and looking for ideas from both internal and external sources.
+                </p>
+              </div>
+            </article>
+            <article class="program-container">
+              <div class="left-text">
+                <img class="leficon" src="./image/program_icon_01.png" alt="lecture icon" />
+                <p class="leftext">The New Product Innovation</p>
+              </div>
+              <div class="right-text">
+                <p class="rightext">
+                  You will learn the art of creating or improving the products in a
+                  way that tries to solve problems for consumers and companies or society at large
+                </p>
+              </div>
+           </article>
+           <article class="program-container design">
+              <div class="left-text">
+                <img class="leficon" src="./image/program_icon_04.png" alt="Workshop icon" />
+                <p class="leftext">Design Thinking</p>
+              </div>
+              <div class="right-text">
+                <p class="rightext">
+                  Understand customer needs and problem-solve creatively in order to meet them
+                </p>
+              </div>
+           </article>
```

Comment:
- [ ] [OPTIONAL] Amazing job implementing the required changes. I think the white border around the card is not supposed to remain static on one card. I think it is indicating the hover state on all cards. It would be great if the white border appears around a card when hovering over it. 😎️

![image](https://user-images.githubusercontent.com/17243325/217504678-edd5e8d6-7f69-4822-b50a-43bd48fb8c27.png)